industrial_ci/src/tests/source_tests.sh

```diff
@@ -23,6 +23,8 @@
 # (As of version 0.4.4 most of them are defined in env.sh).
 
 function install_catkin_lint {
+    ici_apt_install software-properties-common
+    ici_asroot apt-add-repository ppa:roehling/ros -y -u # -u for versions pre-Ubuntu 18.04
```

Comment:
This could be made a function (in workspace.sh): `ici_add_apt_repo ppa:roehling/ros`
This would even work from any hook script.